server/src/game/index.js

```diff
@@ -99,6 +102,35 @@ export class Game {
     this.log(`will start in ${env.START_GAME_TIMEOUT} seconds`);
   }
 
+  spawnMobs() {
+    const getRandomSpawnPoint = (map) => {
+      let spawnPoint;
+
+      while (!spawnPoint) {
```

Comment:
inefficient, it would be safer to iterate on the map, and then spawn a mob with a low probability on each empty cell

Reply:
Ok, but do you really want me to take time on this other than progress in the project ?

Reply:
whatever